### DISCUSSION TITLE
A widget specifically for managing todo

### DISCUSSION THREAD

**Discussion by mendhak** [gsantner/markor#2263] [Ideas] [answered]
Hi, wanted to share an idea.  

Since the 'todo' file is a format known to the app, there could be a widget just for the todo. It shows the pending/completed todo items, with checkboxes against them (done/not done). There could be a little `+` in the widget that opens an input dialog, takes the next todo, and adds it to the file. 

![image](https://github.com/gsantner/markor/assets/746276/943aba4b-30fb-4509-9c54-05afa4027a1c)

**Comment by gsantner** [comment, marked as answer]
#156 , #1997
